### PR TITLE
VACMS-7735: add method "alias" to allow workbench_menu_access check t…

### PR DESCRIPTION
**Pull request by timcosgrove**
See #7735

### CODE REVIEW

**Comment by indytechcook** [comment]
Looks like Worbench should be called `access` since that's what is on the interface.  `access` does call `checkAccess` but all public interaction with the `AccessHandlerInterface` should go through `access`.

```
  /**
   * Checks access to an operation on a given entity or entity translation.
   *
   * Use \Drupal\Core\Entity\EntityAccessControlHandlerInterface::createAccess()
   * to check access to create an entity.
   *
   * @param \Drupal\Core\Entity\EntityInterface $entity
   *   The entity for which to check access.
   * @param string $operation
   *   The operation access should be checked for.
   *   Usually one of "view", "view label", "update" or "delete".
   * @param \Drupal\Core\Session\AccountInterface $account
   *   (optional) The user session for which to check access, or NULL to check
   *   access for the current user. Defaults to NULL.
   * @param bool $return_as_object
   *   (optional) Defaults to FALSE.
   *
   * @return bool|\Drupal\Core\Access\AccessResultInterface
   *   The access result. Returns a boolean if $return_as_object is FALSE (this
   *   is the default) and otherwise an AccessResultInterface object.
   *   When a boolean is returned, the result of AccessInterface::isAllowed() is
   *   returned, i.e. TRUE means access is explicitly allowed, FALSE means
   *   access is either explicitly forbidden or "no opinion".
   */
  public function access(EntityInterface $entity, $operation, AccountInterface $account = NULL, $return_as_object = FALSE);
```

File: docroot/modules/custom/va_gov_graphql/src/Access/MenuLinkContentAccessHandler.php [hunk 1]
```diff
@@ -37,4 +38,21 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
...
+   * @param string $operation
+   *   The operation being performed.
+   * @param \Drupal\Core\Session\AccountInterface $account
+   *   The user account for which access is being checked.
+   *
+   * @return \Drupal\Core\Access\AccessResultInterface
+   *   The access result.
+   */
+  public function accessCheck(EntityInterface $entity, $operation, AccountInterface $account) : AccessResultInterface {
+    return $this->checkAccess($entity, $operation, $account);
```
Comment (indytechcook) [inline docroot/modules/custom/va_gov_graphql/src/Access/MenuLinkContentAccessHandler.php:55]:
```suggestion
    return $this->access($entity, $operation, $account);
```

**Review by swirtSJW — COMMENTED** [review]
Left some comments in the issue.  This approach doesn't seem right to me.

**Comment by ndouglas** [comment]
Gonna close this since the OP issue was closed.

**Pull request closed without merging**